package.json

```diff
@@ -50,27 +50,27 @@
     "react-native-keyboard-aware-scrollview": "^2.0.0"
   },
   "devDependencies": {
+    "@babel/core": "7.4.5",
     "@storybook/addon-actions": "3.3.15",
     "@storybook/addon-storyshots": "3.3.15",
     "@storybook/react-native": "3.3.15",
     "babel-cli": "6.26.0",
-    "babel-eslint": "8.2.1",
+    "babel-eslint": "10.0.1",
     "babel-plugin-module-resolver": "3.0.0",
     "babel-preset-react-native": "4.0.0",
     "chokidar": "2.1.2",
-    "eslint": "4.17.0",
-    "eslint-config-prettier": "3.3.0",
-    "eslint-plugin-instawork": "0.0.2",
-    "eslint-plugin-prettier": "3.0.0",
+    "eslint": "5.16.0",
```

Comment:
Are there any breaking changes in this eslint upgrade that may require changes in the custom rules in `eslint-plugin-instawork`?

Reply:
Not that I'm aware of - `5.16.0` is the same version used by `eslint-plugin-instawork`